### PR TITLE
changed formatting

### PR DESCRIPTION
**Pull request by SDToews**
# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/api/common/fetchAPI.js [hunk 1]
```diff
@@ -1,43 +1,42 @@
...
-        } else {
-            if (isDelete) return
+  const response = await fetch(url, options);
 
-            const data = await response.json()
-            return responseTransform(data)
-        }
+  if (response.ok) {
+    if (response.status === HTML_RESPONSE.NO_CONTENT) {
+      return;
```
Comment (akroon3r) [inline src/api/common/fetchAPI.js:26]: I think we should return a 204 here?

File: src/api/common/fetchAPI.js [hunk 2]
```diff
@@ -1,43 +1,42 @@
...
-        console.warn('response', response)
-        return []
+      if (isDelete) return;
+
+      const data = await response.json();
+      return responseTransform(data);
     }
-}
+  } else {
+    console.error(`${response.status} ${response.statusText} for ${url}`);
```
Comment (akroon3r) [inline src/api/common/fetchAPI.js:38]: same with these

Comment (akroon3r) [reply src/api/common/fetchAPI.js:38]: line 38,39

File: src/hooks/useLicensees/useLicensees.js [hunk 3]
```diff
@@ -19,92 +19,129 @@ import { licenseeFields } from './licenseeFields'
...
+  let formColumns = 1;
+
+  const softwareTable = useSoftware();
+
+  const licenseeTable = useDBTableFactory('licensee', licenseeId);
+
+  const data = useMemo(() => {
+    if (
+      licenseeTable.isLoading ||
+      licenseeTable.isError ||
```
Comment (akroon3r) [inline src/hooks/useLicensees/useLicensees.js:31]: weird newline or carriage return here

File: src/hooks/useSoftware/useSoftware.js [hunk 4]
```diff
@@ -1,28 +1,39 @@
...
-
-    const create = (body) => {
-        const renewal = body.renewal === '' ? null : body.renewal
-        return softwareTable.create({ ...body, renewal })
-    }
-
-    return { ...softwareTable, data, create, tableColumns, formFields, formColumns }
-}
\ No newline at end of file
+  let formColumns = 1;
```
Comment (akroon3r) [inline src/hooks/useSoftware/useSoftware.js:7]: I feel like we can do lines 7 and 9 a bit more cleanly. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

might be the solution

**Review by akroon3r — CHANGES_REQUESTED** [review]
A few things to review before merging.

File: src/hooks/useLicensees/useLicensees.js (hunk 3 above)
Comment (SDToews) [reply src/hooks/useLicensees/useLicensees.js:31]: indeed, on my current branch, locally, it is all on one line

**Comment by SDToews** [comment]
Since the purpose of this pull request / branch is to update the formatting, I think we should leave it and raise a tickets to deal with the issues.

I've created IOD-207 to address the issues

**Review by akroon3r — APPROVED** [review]

**Pull request merged**